jwst/refpix/irs2_subtract_reference.py

```diff
@@ -194,14 +195,28 @@ def correct_model(output_model, irs2_model, scipix_n_default=16, refpix_r_defaul
 
 
 def float_to_complex(data):
-    """Convert real and imaginary parts to complex"""
+    """
+    Convert real and imaginary parts to complex.
+
```

Comment:
missing parameters section?  (wondering why numpydoc didn't catch this)

Reply:
Added